src/main/java/net/masterthought/cucumber/json/support/Status.java

```diff
@@ -38,8 +38,15 @@ public String getLabel() {
         return String.valueOf(name().charAt(0)).toUpperCase() + name().substring(1).toLowerCase();
     }
 
-    public String toHtmlClass() {
+    public String toHtmlClass(String name) {
         // TODO: should be moved out of this class
-        return "<div class=\"" + this.getName().toLowerCase() + "\">";
+        String uniqueIdentifier = name.trim().toLowerCase().replaceAll(" ", "");
+        return "<div class=\"" + this.getName().toLowerCase() + "\" cuc=\""+uniqueIdentifier+"\">";
```

Comment:
i guess it shouldn't be called `name` but something like `parentUUID` all steps will look like that:
`<div class="passed" cuc="uuIdOfScenarioOrBackground"> (...) </div>`